stencil-library/.babelrc.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Is babel necessary? I don't see how it's being used but I would like to avoid introducing build tools that don't come with Stencil or Storybook out of the box

Reply:
https://storybook.js.org/docs/react/configure/babel

This was a result of migration to the latest version. It automatically created a default babelrc file to keep a babel config somewhere. It was already being used, but the config file was hidden in previous versions.

Reply:
Okay, awesome 👍